templates/main-box.tpl.php

```diff
@@ -7,6 +7,11 @@
 
 ?>
 <div id="wpbdp-main-box" class="wpbdp-main-box" data-breakpoints='{"tiny": [0,360], "small": [360,560], "medium": [560,710], "large": [710,999999]}' data-breakpoints-class-prefix="wpbdp-main-box">
+<?php
+    if ( isset( $_GET['inactive_listing'] ) && (int) $_GET['inactive_listing'] === 1 ) {
+	echo wpbdp_render_msg( _x( 'The listing you are trying to access is currently inactive or no longer available.', 'inactive-listing', 'business-directory-plugin' ), 'notice' );
```

Comment:
Can we remove the 'inactive-listing' part here? I don't think it's helpful in this case so it would be nice to keep the code more simple.

Reply:
Do you mean the context property of the _x function? That will result in a phpcs error because it is a [required parameter](https://developer.wordpress.org/reference/functions/_x/) of the function.

Or do you mean the query parameter as a whole? I guess we could change the approach to a transient, but I opted for the query parameter to avoid the cache/database calls.

Reply:
Sorry. I meant switch _x() to __() since the extra description isn't helpful to a translator. I frequently make this switch in BD because it ends up creating a bunch of extra strings that need translation when they are used in the same way.